tests/e2e/cucumber/steps/ui/application.ts

```diff
@@ -37,8 +39,21 @@ When(
 When(
   '{string} opens the {string} url',
   async function (this: World, stepUser: string, url: string): Promise<void> {
-    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    let page: Page
+    try {
+      page = this.actorsEnvironment.getActor({ key: stepUser }).page
+    } catch {
+      page = (
+        await this.actorsEnvironment.createActor({
+          key: stepUser,
```

Comment:
if you do not want to check for public users then you can directly use `createActor` here as well